src/tests/unit/tests/reports/components/report-sections/footer-text.test.tsx

```diff
@@ -1,19 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { EnvironmentInfo } from 'common/environment-info-provider';
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 import { FooterText } from 'reports/components/report-sections/footer-text';
 
 describe('FooterText', () => {
     it('renders', () => {
-        const environmentInfo: EnvironmentInfo = {
-            extensionVersion: '1.2.3',
-            browserSpec: 'dummy browser version 1.0.1',
-            axeCoreVersion: '4.5.6',
-        };
+        const environmentInfoProvider = new EnvironmentInfoProvider(
+            '1.2.3',
+            'dummy browser version 1.0.1',
+            '4.5.6',
+        );
 
-        const footerWrapper = shallow(<FooterText {...{ environmentInfo }} />);
+
+
+        const footerWrapper = shallow(<FooterText {...{ environmentInfoProvider }} />);
```

Comment:
nitpick: this could be a little nicer by creating a props object:
```ts
const props: FooterTextProps = {
    environmentInfoProvider,
};

const footerWrapped = shallow(<FooterText {...props} />);
```
This way, the next time we change the `FooterText`'s properties, the line where we shallow render it will not change.